api/src/main/resources/workbench.yaml

```diff
@@ -501,18 +501,24 @@ paths:
 
   # Notebook clusters ####################################################################
 
-  /v1/clusters:
+  /v1/clusters/{clusterNamespace}:
     get:
       summary: List available notebook clusters
       description: >
-        Returns the clusters available to the current user. Currently there is a
-        single default cluster supported per researcher and this cluster should
-        always either exist or be in the process of being initialized. In a
-        future where researchers have more control over cluster creation, this
-        endpoint would be extended to return all clusters.
+        Returns the clusters available to the current user in the clusterNamespace.
+        Currently there is a single default cluster supported per clusterNamespace
+        and this cluster should always either exist or be in the process of being
+        initialized. In a future where researchers have more control over cluster
+        creation, this endpoint would be extended to return all clusters.
       operationId: listClusters
       tags:
         - cluster
+      parameters:
+        - in: path
+          name: clusterNamespace
+          description: clusterNamespace
```

Comment:
Whatever we end up landing on for naming of this parameter, please include a slightly longer description than a re-statement of the param name :)